test/html/Test-Game-UUID.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Test Game</title>
+<script>
+/* UUID://448E73DF-2D2F-47E7-A494-A46B40D4CFB5// */
```

Comment:
As a nitpick, IFWiki recommended an HTML comment, `<!-- UUID://XXXXXXXXXXXXXXXX// -->` not a `<script>` JS comment. But either one works.